tools/buildtools/packmol/packmol.xml

```diff
@@ -69,11 +69,11 @@ with open("$parameteroutfile",'w+') as f:
                 <expand macro="multiple_input" name="packmol_inputs" format="moldy" label="moldy input with molecule type" help_text="Select a moldy file in .moldy format"/>
             </when-->
         </conditional>
-        <section name="allparams" title="General Parameters" expanded="false">
-            <param name="tolerance" type="float" value="2.0" label="distance tolerance" min="0.0" max="100.0"/>
-            <param name="nloop" type="integer" value="20" label="maximum number of optimisation loops" min="1" max="10000"/>
-            <param name="maxit" type="integer" value="20" label="maximum number of gencan iterations per loop" min="1" max="10000"/>
-            <param name="seed" type="integer" value="-1" label="random number generator seed, default of -1 used the computer time to set the seed" />
+        <section name="allparams" title="General parameters" expanded="false">
```

Comment:
@chrisbarnettster can we remove this conditional for now until/if the other formats are added? It looks a bit strange from a UI perspective.

Reply:
Hi @simonbray 
sure, some minor updates to templates will be needed. I'm happy to leave the commented placeholder code for the future 😄

Reply:
done in https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/pull/35/commits/6fa3234ddf93f4fbda7e54c19a192c8a3c1fe58e